### PR TITLE
feat: add Compendium Roll Table

### PR DESCRIPTION
**Pull request by Caladcholg**
Add to the compendium of lhtrpg:

- Treasure Table
- Exhaustion Table
- Random Musical Instrument Type

And all items linked to the previous Roll Table:

- Valuable
- Magic Reagent
- Core Material

### CODE REVIEW

**Comment by Tenyryas** [comment]
I'll look at it a bit further when I'm at home, but I'll probably need to adapt my import script to adapt to the fact you added a `drop` type to the item types. 

Dropped items are under the generic `gear` type instead in the API, but with the use of tags as conditionals we can probably type them accordingly.

File: src/packs/magic-reagants/Magic_Catalyst_06_S2cugdT1ldGqEAZL.json
Comment (Tenyryas) [inline src/packs/magic-reagants/Magic_Catalyst_06_S2cugdT1ldGqEAZL.json:1]: There seems to be an error in the rank here, nothing that'd break the system but may as well fix it before we push it.

File: src/packs/magic-reagants/Magic_Catalyst_06_S2cugdT1ldGqEAZL.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "name": "Magic Catalyst 06",
+  "type": "drop",
+  "_id": "S2cugdT1ldGqEAZL",
+  "img": "systems/lhtrpg/assets/ui/items_icons/drop.svg",
+  "system": {
+    "description": "",
+    "tags": [
+      "Magic Catalyst 6"
+    ],
+    "rank": 1,
```
Comment (Caladcholg) [inline src/packs/magic-reagants/Magic_Catalyst_06_S2cugdT1ldGqEAZL.json]:
```suggestion
    "rank": 6,
```

File: src/packs/magic-reagants/Magic_Catalyst_06_S2cugdT1ldGqEAZL.json
Comment (Caladcholg) [reply src/packs/magic-reagants/Magic_Catalyst_06_S2cugdT1ldGqEAZL.json:1]: Fixed

**Review by Tenyryas — APPROVED** [review]

**Pull request merged**